tools/ci_connector_ops/ci_connector_ops/pipelines/contexts.py

```diff
@@ -357,6 +360,18 @@ def metadata(self) -> dict:
     def docker_image_from_metadata(self) -> str:
         return f"{self.metadata['dockerRepository']}:{self.metadata['dockerImageTag']}"
 
+    @property
+    def ci_gcs_credentials_secret(self) -> Secret:
+        # TODO (ben): Update this to be in use ANYWHERE we use a service account.
```

Comment:
@alafanechere I didnt want to add yet another service account env var. It felt like we were going down a bad way there.

Im thinking that we should enforce one service account for the entire pipeline. WDYT?